github/github_test.go

```diff
@@ -135,32 +134,26 @@ func testBody(t *testing.T, r *http.Request, want string) {
 	}
 }
 
-// Helper function to test that a value is marshalled to JSON as expected.
+// Test whether the marshalling of v produces a JSON that corresponds
```

Comment:
Let's please spell "marshal" and "unmarshal" consistently throughout the PR with a single "l".
Please also terminate the comment sentence with a period.

Reply:
done

Reply:
(also, added the issue I'm helping with to the description of the PR and the commit message)